website/static/js/profile.js

```diff
@@ -122,6 +103,35 @@ var DateMixin = function() {
         self.endYear('');
         return true;
     };
+
+    self.startYear.extend({
```

Comment:
Does moving this code here have an effect on the behavior? If so, can you explain why? If not, it's more readable to have the `extend` calls in the same place the observables are initialized.